pkg/solana/fees/block_history.go

```diff
@@ -44,6 +54,7 @@ func NewBlockHistoryEstimator(c func(context.Context) (client.ReaderWriter, erro
 		cfg:    cfg,
 		lgr:    lgr,
 		price:  cfg.ComputeUnitPriceDefault(), // use default value
+		cache:  blockMedianCache{medianMap: make(map[uint64]ComputeUnitPrice)},
 	}, nil
```

Comment:
nit: we already know the capacity of the cache. wdyt about preallocating the size?  
```suggestion
		cache:  blockMedianCache{medianMap: make(map[uint64]ComputeUnitPrice, cfg.BlockHistorySize)},
	}, nil
```

Reply:
The capacity probably exceeds the `BlockHistorySize` for a bit before we clean it up but still a good idea to allocate at least that much to avoid unnecessary reallocs. Good catch! I'll also initialize `storedBlockRange`